addons/account/tests/test_multivat.py

```diff
@@ -102,6 +102,7 @@ def test_data_getter(self, template_code):
                 for name, xmlid, amount in (
                     (f'{external_id_prefix}Tax 1', 'test_tax_1_template', 15),
                     (f'{external_id_prefix}Tax 2', 'test_tax_2_template', 0),
+                    ('tax with the same name', 'test_tax_3_template', 0),
```

Comment:
why that?

Reply:
I'd rather see the prefix being removed on the lines above

Reply:
> why that?

I'm not sure to understand the question.
It was to add some data to the test suite. The previous `f'{external_id_prefix}`...` makes the tax being instantiated with different names.
This one would have kept the same name.
I changed it to follow William insight because it's indeed better.